experimental/auth-logic/client-verification/examples/oak_verification_passing.auth_logic

```diff
@@ -0,0 +1,65 @@
+.decl attribute has_expected_hash_from(hash : Sha256Hash, expecter : Principal)
```

Comment:
No that was there a while. In the auth logic language there are relation declarations that give the types of the parameters of a relation, but for the tool in the transparent release repository, the relation declarations are given in one of the wrappers rather than in one of the templates. Since the final output authorization logic policy was not checked into the repo, you probably would not have had a reason to see the relation declarations :)

The attribute keyword as part of a relation declaration just means that this is a predicate that is used as an attribute of a principal.